src/util/heap/layout/mmapper.rs

```diff
@@ -37,6 +37,7 @@ pub trait Mmapper: Sync {
         start: Address,
         pages: usize,
         strategy: MmapStrategy,
+        anno: &MmapAnnotation,
```

Comment:
Need to update the doc comment to mention the new argument

Reply:
Done.